.github/workflows/integration_test.yml

```diff
@@ -45,5 +45,5 @@ jobs:
         TRAVIS_COM_TOKEN: ${{ secrets.TRAVIS_COM_TOKEN }}
         EVENT_MESSAGE: ${{ github.event.message }}
         HOME: 'home/runner'
-      run: |
```

Comment:
This looks like an unnecessary change. can you re add the space here so that it does not appear in the PR diff